.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,16 +34,21 @@ A detailed explanation of this checklist is available here:
 https://github.com/nvaccess/nvda/blob/master/devDocs/githubPullRequestTemplateExplanationAndExamples.md#code-review-checklist
 -->
 
-- [ ] Pull Request description is up to date.
-- [ ] Unit tests.
-- [ ] System (end to end) tests.
-- [ ] Manual testing.
+<<<<<<< HEAD
```

Comment:
A merge conflict artifact

Reply:
Thanks. Fixed in 6e354947cd429d8fae5016ea5222db201de2b0b9.